doc/Syscalls.md

```diff
@@ -312,6 +312,13 @@ On the next `switch_to_user` call, the application will resume execution based
 on the process stack pointer, which points to the instruction after the system
 call that switched execution to the kernel.
 
+Syscalls may clobber userspace memory. The kernel will not clobber any userspace
```

Comment:
Which userspace memory? Do you mean `AppSlice`s (memory shared with the kerne)? Or something else?

Reply:
Frankly, I just read this out of the clobbers listed in the syscall code in tock.c.

I *think* that is referring to memory shared using `allow()`. Let me know if it's some other memory and I'll update the documentation.

Reply:
@ppanuto (?) should know what that memory clobber is for

Reply:
@ppannuto (spelling fix)